kubernetes/docs/V1Probe.md

```diff
@@ -8,7 +8,7 @@ Name | Type | Description | Notes
 **failure_threshold** | **int** | Minimum consecutive failures for the probe to be considered failed after having succeeded. Defaults to 3. Minimum value is 1. | [optional] 
 **grpc** | [**V1GRPCAction**](V1GRPCAction.md) |  | [optional] 
 **http_get** | [**V1HTTPGetAction**](V1HTTPGetAction.md) |  | [optional] 
-**initial_delay_seconds** | **int** | Number of seconds after the container has started before liveness probes are initiated. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes | [optional] 
+**initial_delay_seconds** | **int** | Number of seconds after the container has started before liveness probes are initiated. Defaults to 0. Minimum value is 0. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes | [optional] 
```

Comment:
This documentation is generated based on the [upstream openapi spec](https://github.com/search?q=repo%3Akubernetes%2Fkubernetes%20%22Number%20of%20seconds%20after%20the%20container%20has%20started%20before%20liveness%20probes%20are%20initiated%22&type=code). Please fix the documentation upstream.

/close